src/test/java/org/gridsuite/modification/server/modifications/LineAttachToVoltageLevelTest.java

```diff
@@ -42,12 +44,19 @@ private LineCreationInfos getAttachmentLine(String lineName) {
 
     private VoltageLevelCreationInfos getNewVoltageLevel() {
         return VoltageLevelCreationInfos.builder()
-                .equipmentId("newVlName")
+                .equipmentId("newVoltageLevel")
                 .equipmentName("NewVoltageLevel")
                 .nominalVoltage(379.3)
                 .substationId("s1")
-                .busbarSections(List.of(new BusbarSectionCreationInfos("v1bbs", "BBS1", 1, 1)))
-                .busbarConnections(List.of())
+                .nominalVoltage(379.1)
```

Comment:
useless ... nominalVoltage is set just above ...

Reply:
Done